react/week-2/lesson.md

```diff
@@ -429,6 +437,57 @@ To help us cleanly split up code that performs business logic from code that sho
 
 Container components usually have some state and handler methods. Because of this they must use the `class` syntax. Presentational components on the other hand don't require the more verbose syntax. Instead they usually use the functional syntax.
 
+## Fetching data in React
+
+Often when you create a React app, you will want to get data from an API, and display it inside your components.
+How do we do this in React? Where does the API call go, and when should we trigger it?
+
+**Where:** Most likely at the top of the component tree, inside a 'container' component as explained above. You can then flow the data down into your child components as props.
```

Comment:
FYI I'd like to add an image here to illustrate the 'component tree'.

Reply:
👍 Just making a note that this is similar to the existing [Container Components](https://syllabus.codeyourfuture.io/react/week-2/lesson.html#container-components) section.